Jenkinsfile_CNP

```diff
@@ -1,7 +1,6 @@
 #!groovy
 
```

Comment:
remove enableDockerBuild() from this file

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-user-profile-api/pull/252/commits/346362c6450c88462f59bf41431bfe83d48238ac